src/lib/formRegex.ts

```diff
@@ -0,0 +1,13 @@
+// at least 2 characters followed by a space and at least another 2 characters
+// p{L}/u matches any unicode letter
+// this allows for both extended latin, cyrillic and other unicode letter characters
+export const nameRegex = /^\p{L}{2,}\s\p{L}{2,}$/u;
```

Comment:
🟢  Great choice for handling Unicode characters  and specifying the minimum length requirement for both parts of the name.